modules/core_project_factory/variables.tf

```diff
@@ -97,6 +98,11 @@ variable "credentials_path" {
   default     = ""
 }
 
+variable "impersonate_service_account" {
+  description = "An optional service account to impersonate. If this service account is not specified, Terraform will fall back to credential file or Application Default Credentials."
```

Comment:
Should we note this option conflicts with specifying a `credentials_path` to project factory?

Reply:
Yep agreed - I will update this